lib/utils/screen-manager.js

```diff
@@ -12,7 +14,19 @@ function height(content) {
 }
 
 function width(content) {
-  return stripAnsi(content).length;
+  var chars = stripAnsi(content);
+  var len = 0;
+
+  for (var i = chars.length - 1; i >= 0; i--) {
```

Comment:
Let's drop the custom code in favor a module doing what we want - in this case https://www.npmjs.com/package/string-length will do both astral points and ANSI.